sql/parse/parse.go

```diff
@@ -2673,7 +2709,7 @@ func jsonTableExpr(ctx *sql.Context, t *sqlparser.JSONTableExpr) (sql.Node, erro
 		paths[i] = col.Type.Path
 	}
 
-	sch, err := TableSpecToSchema(ctx, t.Spec)
+	sch, err := TableSpecToSchema(ctx, t.Spec, sql.Collation_Default)
```

Comment:
TODO here?

Reply:
JSON tables aren't writeable, so it didn't matter which collation it received as it would not be applied to any columns. This has been changed along with the function change though.